src/auth/auth.spec.ts

```diff
@@ -139,13 +139,17 @@ describe('FirebaseAuth', () => {
       });
     authSpy['getRedirectResult'].and.returnValue(Promise.resolve(null));
 
-    inject([FirebaseApp, AngularFireAuth], (_app: firebase.app.App, _afAuth: AngularFireAuth) => {
-      app = _app;
-      afAuth = _afAuth;
-      authData = null;
-      authCb = null;
-      backend = new FirebaseSdkAuthBackend(app);
-    })();
+    var mockNgZone = Zone.current.fork({
```

Comment:
Could you add a comment on why you need to wrap the code in a zone. What exactly needs to be wrapped ?